rotkehlchen/tests/api/test_data_purging.py

```diff
@@ -17,6 +17,7 @@
 @pytest.mark.parametrize('added_exchanges', [(Location.BINANCE, Location.POLONIEX)])
 def test_purge_all_exchange_data(rotkehlchen_api_server_with_exchanges, added_exchanges):
     rotki = rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen
+    added_exchanges += (Location.FTX,)  # Also check that data for dead exchanges is purged
```

Comment:
nitpick: I would use 

```
exchange_locations = added_exchanges + (Location.FTX,)
```

and then use `exchange_locations` because it makes clear the difference between exchange with instances and without

Reply:
+1. Please do the nitpick